lib/internal/per_context/primordials.js

```diff
@@ -175,5 +175,18 @@ primordials.SafePromise = makeSafe(
   copyPrototype(original.prototype, primordials, `${name}Prototype`);
 });
 
+// Create copies of abstract intrinsic objects that are not directly exposed
+// on the global object.
+// Refs: https://tc39.es/ecma262/#sec-%typedarray%-intrinsic-object
+[
+  { name: 'TypedArray', original: Reflect.getPrototypeOf(Uint8Array) },
+].forEach(({ name, original }) => {
+  primordials[name] = original;
```

Comment:
I don't think we want to have a `primordials.TypedArray`, only the prototype methods are interesting.
```suggestion
```

Reply:
Actually, this is needed by `freeze_intrinsics.js`.